tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,46 @@
+metadata:
```

Comment:
It would be a good idea to include this in one of the `tests/WE2E/machine_suites/coverage.*` test suites and in the `tests/WE2E/machine_suites/comprehensive` test suites, so that this work can be tested regularly for PRs.

Reply:
@MichaelLueken - thanks for the note, I'll add the changes to the comprehensive suite. Also, the plan was to have a 9-h forecast for this test, which covers timing for a specific event. It is running right now with these changes, and I'll add them in the next commit. 

Working on the *rst documentation files at the moment.

Reply:
Sounds good, @natalie-perlin!

Reply:
All done. The 9-h test has been completed successfully as well:
```
Took 0:38:58.580259; will no longer monitor.
All 1 experiments finished
Calculating core-hour usage and printing final summary
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta_  COMPLETE             170.49
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             170.49
```

If this looks too long, we could revert to 3-h test on CONUS domain, but just keep the data for further testing with smaller regional domain.